### PR TITLE
Some documentation updates.

### PR DESCRIPTION
**Pull request by dlorenc**
Add a section on installation and usage.

### CODE REVIEW

**Comment by dlorenc** [comment]
Note that this won't fully work until #21 is merged.

File: README.md [hunk 1]
```diff
@@ -4,7 +4,90 @@ Supply Chain Security in Tekton Pipelines
...
+gpg: Signature made Mon Jul 27 14:42:01 2020 CDT
+gpg:                using RSA key 4FCFD41D993B806C
+gpg: checking the trustdb
+gpg: marginals needed: 3  completes needed: 1  trust model: pgp
+gpg: depth: 0  valid:   4  signed:   0  trust: 0-, 0q, 0n, 0m, 0f, 4u
+gpg: next trustdb check due at 2022-03-16
+gpg: Good signature from "Tekton Unit Tests <testing@tekton.dev>" [ultimate]
+```
+
+Now we have a verifiable record of the TaskRun!s
```
Comment (jonjohnsonjr) [inline README.md:89]:
```suggestion
Now we have a verifiable record of the TaskRuns!
```

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ You must install these tools:
 ## Kubernetes cluster
 
 Docker for Desktop using an edge version has been proven to work for both
-developing and running Pipelines. Your Kubernetes version must be 1.11 or later.
+developing and running Chains. Your Kubernetes version must be 1.11 or later.
 
 To setup a cluster with GKE:
```
Comment (jonjohnsonjr) [inline DEVELOPMENT.md:54]: The knative links 404

File: README.md [hunk 3]
```diff
@@ -4,7 +4,90 @@ Supply Chain Security in Tekton Pipelines
...
-Stay tuned for getting started information!
+To use Tekton Chains, you'll have to build and deploy your own release from git.
+
+### Installation
+
+See [DEVELOPMENT.md](DEVELOPMENT.md) for a guide on how to build and deploy your own version.
+
+### Configuration
+
+#### Signing Secrets
```
Comment (jonjohnsonjr) [inline README.md:15]: I like newlines after section headings.

File: README.md [hunk 4]
```diff
@@ -4,7 +4,90 @@ Supply Chain Security in Tekton Pipelines
...
+There are many ways to go about this, but you can usually use something like this:
+
+```shell
+gpg --gen-key
+```
+
+Enter a passprase (make sure you remember it!) and a name for the key.
+
+Next, you'll need to upload the private key as a Kubernetes `Secret` so Tekton can use it
+to sign.
```
Comment (jonjohnsonjr) [inline README.md:27]: to sign what?

File: README.md [hunk 5]
```diff
@@ -4,7 +4,90 @@ Supply Chain Security in Tekton Pipelines
...
+
+```shell
+kubectl delete secret signing-secrets -n tekton-pipelines && kubectl create secret generic signing-secrets -n tekton-pipelines --from-file=pgp.passphrase --from-file=pgp.private-key --from-file=pgp.public-key
+```
+
+## Usage
+
+In its default mode of operation, Chains works by observing all `TaskRuns` in your cluster.
+When `TaskRuns` complete, Chains takes a snapshot of them.
+Chains then converts this snapshot to one or more standard payload formats, signs those and stores them somewhere.
```
Comment (jonjohnsonjr) [inline README.md:51]:
```suggestion
Chains then converts this snapshot to one or more standard payload formats, signs them, and stores them somewhere.
```

File: DEVELOPMENT.md (hunk 2 above)
Comment (dlorenc) [reply DEVELOPMENT.md]: Fixed

**Comment by dlorenc** [comment]
Thanks for the review. Changed the knative URL to just point to the general one in tekton pipeline.

**Pull request merged**